packages/@sanity/vision/src/components/QueryRecall.tsx

```diff
@@ -0,0 +1,430 @@
+                    ) : (
+                      <Text
+                        weight="bold"
+                        size={3}
+                        textOverflow="ellipsis"
+                        style={{maxWidth: '170px', cursor: 'pointer', padding: '4px 0'}}
+                        title={
+                          optimisticTitles[q._key] ||
+                          q.title ||
+                          q._key.slice(q._key.length - 5, q._key.length)
+                        }
+                        onClick={() => {
+                          setEditingKey(q._key)
+                          setEditingTitle(q.title || q._key.slice(0, 5))
+                        }}
+                      >
+                        {optimisticTitles[q._key] ||
+                          q.title ||
+                          q._key.slice(q._key.length - 5, q._key.length)}
+                      </Text>
+                    )}
+                    {isEdited && (
+                      <Box
+                        style={{
+                          width: '6px',
+                          height: '6px',
+                          borderRadius: '50%',
+                          backgroundColor: 'var(--card-focus-ring-color)',
+                        }}
+                      />
+                    )}
+                  </Flex>
+                  <MenuButton
+                    button={<EllipsisVerticalIcon />}
+                    id={`${q._key}-menu`}
+                    menu={
+                      <Menu
+                      // style={{background: 'white', backgroundColor: 'white', borderRadius: '11%'}}
+                      >
+                        <MenuItem
+                          padding={0}
+                          radius={3}
+                          style={{background: 'transparent', backgroundColor: 'transparent'}}
+                        >
+                          <Button
+                            mode="bleed"
+                            style={{background: 'transparent', backgroundColor: 'transparent'}}
+                            tone="critical"
+                            width="fill"
+                            onClick={(event) => {
+                              event.stopPropagation()
+                              deleteQuery(q._key)
+                            }}
+                            disabled={deleting.includes(q._key)}
+                          >
+                            <Flex align="center" gap={2} padding={1}>
+                              <Box
+                                style={{fontSize: '1.25em', display: 'flex', alignItems: 'center'}}
+                              >
+                                <TrashIcon />
+                              </Box>
+                              <Text size={1}>{t('action.delete')}</Text>
+                            </Flex>
+                          </Button>
+                        </MenuItem>
+                      </Menu>
```

Comment:
sorry. this is not super clear from how the components are implemented, but this could be replaced by 
```suggestion
        <MenuItem
                            tone="critical"
                            padding={3}
                            icon={TrashIcon}
                            text={t('action.delete')}
                            onClick={(event) => {
                              event.stopPropagation()
                              deleteQuery(q._key)
                            }}
                          />
```

That will make it align with the rest of the studio menu items.